presto-common/src/main/java/com/facebook/presto/common/predicate/TupleDomainFilter.java

```diff
@@ -249,7 +249,7 @@ public boolean testLength(int length)
         @Override
         public String toString()
         {
-            return toStringHelper(this).toString();
+            return "AlwaysFalse{}";
```

Comment:
Let's use `format("%s{}", this.getClass().getName())` instead? Just in case the class is renamed in the future.